occupi-backend/pkg/mail/mail.go

```diff
@@ -52,3 +54,65 @@ func SendMultipleEmailsConcurrently(emails []string, subject, body string) []str
 
 	return emailErrors
 }
+
```

Comment:
@Rethakgetse-Manaka , I noticed that you are still sending emails using concurrent methods, which we had highlighted might slow api response times significantly if say an email was to be sent out to 1000 people.
Did you consider using bcc to send the email to multiple people. If it did not work, could I suggest you creating some other branch and writing unit tests to ensure this function is stable, and doesn't get blocked by google if we say try to send many emails in seconds

Reply:
I'll do that